spec/rubocop/cop/style/method_call_with_args_parentheses_spec.rb

```diff
@@ -468,6 +468,37 @@ def method_missing(name, ...)
             end
         RUBY
       end
+
+      it 'registers an offense in case_match multi-line branches' do
+        expect_offense(<<~RUBY)
+          case match
+          in :pattern1
+            foo(value:)
+               ^^^^^^^^ Omit parentheses for method calls with arguments.
+          in :pattern2
+            bar(value:)
+               ^^^^^^^^ Omit parentheses for method calls with arguments.
+          end
+        RUBY
```

Comment:
Can you add `expect_corrections`?

```suggestion
        expect_offense(<<~RUBY)
          case match
          in :pattern1
            foo(value:)
               ^^^^^^^^ Omit parentheses for method calls with arguments.
          in :pattern2
            bar(value:)
               ^^^^^^^^ Omit parentheses for method calls with arguments.
          end
        RUBY

        expect_correction(<<~RUBY)
          case match
          in :pattern1
            foo value:
          in :pattern2
            bar value:
          end
        RUBY
```